### PR TITLE
doc(README): initialize readme content

### PR DESCRIPTION
**Pull request by akantcheff**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,188 @@
...
+    * Layouts
+    * Living Applications
+    * Organizations
+    * Pages
+    * Processes
+    * Profiles **(Subscription only)**
+    * REST API Extensions
+    * Themes
+* **(Subscription only)** The *Application Archive* generated with [Bonita Continuous Delivery](https://documentation.bonitasoft.com/bcd/latest/livingapp_build) is a proper candidate for a custom application.
+* **(Subscription only)** The *Application Configuration* (*.bconf* file) generated with [Bonita Continuous Delivery](https://documentation.bonitasoft.com/bcd/latest/livingapp_build) is an extra deployable artifact compatible with this tool.
```
Comment (educhastenier) [inline README.md:28]:
```suggestion
* **(Subscription only)** The *Application Configuration Archive* (*.bconf* file) generated with [Bonita Continuous Delivery](https://documentation.bonitasoft.com/bcd/latest/livingapp_build) is an extra deployable artifact compatible with this tool.
```

File: main.go [hunk 2]
```diff
@@ -35,7 +35,7 @@ const (
 var (
 	// Flags:
 	tomcatFlag = flag.Bool("tomcat", false, `Choose to build a Bonita Tomcat bundle containing your application
-use -bonita-tomcat-bundle to specify the path (on your machine) to the Bonita tomcat bundle file (Bonita*.zip); otherwise the file is looked for in the current folder`)
```
Comment (educhastenier) [inline main.go:38]: Why removing this?

File: README.md [hunk 3]
```diff
@@ -1 +1,188 @@
-# Bonita Application Packager
\ No newline at end of file
+# Bonita Application Packager tool
+
+The Bonita Application Packager tool is provided by Bonitasoft to allow you to build a Bonita Tomcat bundle or a Bonita Docker image containing your custom application.
```
Comment (educhastenier) [inline README.md:3]:
```suggestion
The Bonita Application Packager tool is provided by Bonitasoft to allow you to build a Bonita Tomcat bundle or a Bonita Docker image containing your custom application.
The advantage of such a packaging is that Bonita automatically installs your entire application (pages, Rest API extensions, themes, BDM, ...) at startup, relieving you from having to install all artifact by hand one by one.
```

File: README.md [hunk 4]
```diff
@@ -1 +1,188 @@
...
+    * BDM Access Control **(Subscription only)**
+    * Layouts
+    * Living Applications
+    * Organizations
+    * Pages
+    * Processes
+    * Profiles **(Subscription only)**
+    * REST API Extensions
+    * Themes
+* **(Subscription only)** The *Application Archive* generated with [Bonita Continuous Delivery](https://documentation.bonitasoft.com/bcd/latest/livingapp_build) is a proper candidate for a custom application.
```
Comment (educhastenier) [inline README.md:27]: Great to point to BCD doc ! 👍🏽

File: README.md [hunk 5]
```diff
@@ -1 +1,188 @@
...
+
+* Basic Community usage:
+
+```
+bonita-application-packager -docker /path/to/my-custom-application.zip
+```
+
+The result is a Docker image named `bonita-application-community`.
+
+Per default, the Bonita based image used is `bonita:latest`, located on [DockerHub](https://hub.docker.com/_/bonita).
```
Comment (educhastenier) [inline README.md:138]:
```suggestion
By default, the Bonita based image used is `bonita:latest`, located on [DockerHub](https://hub.docker.com/_/bonita).
```

File: README.md [hunk 6]
```diff
@@ -1 +1,188 @@
...
+
+* Basic Subscription usage:
+
+```
+bonita-application-packager -docker -subscription -registry-username my-username -registry-password my-password /path/to/my-custom-application.zip 
+```
+
+The result is a Docker image named `bonita-application-subscription`.
+
+Per default, the Bonita based image used is `quay.io/bonitasoft/bonita-subscription:latest`.
```
Comment (educhastenier) [inline README.md:149]:
```suggestion
By default, the Bonita based image used is `quay.io/bonitasoft/bonita-subscription:latest`.
```

**Review by educhastenier — CHANGES_REQUESTED** [review]
Very nice piece of work 👍🏽 
Just a few remarks

File: main.go (hunk 2 above)
Comment (akantcheff) [reply main.go:38]: I think it is redundant and we don't precise this on the other options.

**Review by educhastenier — APPROVED** [review]

**Pull request merged**